Source/WebInspectorUI/UserInterface/Views/ConsoleDrawer.js

```diff
@@ -148,13 +148,13 @@ WI.ConsoleDrawer = class ConsoleDrawer extends WI.ContentBrowser
 
     _restoreDrawerHeight()
     {
-        this._updateDrawerHeight(this._drawerHeightSetting.value);
+        this._updateDrawerHeight(this._drawerHeightSetting.value, WI.quickConsole.element.offsetHeight);
     }
 
-    _updateDrawerHeight(height)
+    _updateDrawerHeight(height, quickConsoleHeight)
     {
         const minimumHeight = 64;
-        const maximumHeight = this.element.parentNode.offsetHeight - WI.TabBrowser.MinimumHeight - WI.QuickConsole.MinimumHeight;
```

Comment:
i dont think `WI.QuickConsole.MinimumHeight` is used anywhere else, so can we remove it?

or maybe we need to `Math.max(quickConsoleHeight, WI.QuickConsole.MinimumHeight)` instead (or more ideally before it's even passed to `_updateDrawerHeight` so that we don't have to recalculate it each time)?  if not, do we also need to update any of the uses of `--console-prompt-min-height`?